.pre-commit-config.yaml

```diff
@@ -27,11 +27,11 @@ repos:
         pass_filenames: true
         args: ["--profile", "black"]
 
-  - repo: https://github.com/ambv/black
-    rev: 20.8b1
-    hooks:
-      - id: black
-        language_version: python3.8
+  #  - repo: https://github.com/ambv/black
```

Comment:
Oops, was this by mistake?

Reply:
No, it was failing on a weird error and I wanted to prioritize shipping this fix over debugging it. Will follow up with a real fix.